v1-0/learn/quick-tour.md

```diff
@@ -104,7 +104,7 @@ The complete source code should look similar to the following:
 import ballerina/http;
 import ballerina/io;
 
-public function main() returns error? {
+public function main() returns @tainted error? {
```

Comment:
Not requiring the `@tainted` annotation here was a bug in pre-1.1.0 versions. So for 1.0.0, without tainted should work. IMO, we shouldn't change this since it could be confusing as to why we need this even though it works without this also.

Reply:
removed the fixes from V1-0.